NitroxClient/GameLogic/SeamothModulesEvent.cs

```diff
@@ -0,0 +1,55 @@
+﻿using NitroxClient.Communication.Abstract;
+using NitroxClient.GameLogic.Helper;
+using NitroxModel.Logger;
+using NitroxModel.Packets;
+using UnityEngine;
+
+namespace NitroxClient.GameLogic
+{
+    public class SeamothModulesEvent
+    {
+        private readonly IPacketSender packetSender;
+
+        public SeamothModulesEvent(IPacketSender packetSender)
+        {
+            this.packetSender = packetSender;
+        }
+
+        public void Update(TechType techType, int slotID, SeaMoth instance)
```

Comment:
Can we split this method into two functions?  That would make it cleaner.

1) BroadcastTorpedoLaunch
2) BroadcastElectricalDefense